src/pt/sussyscan/src/eu/kanade/tachiyomi/extension/pt/sussyscan/SussyToons.kt

```diff
@@ -202,6 +208,30 @@ class SussyToons : HttpSource(), ConfigurableSource {
             Page(index, imageUrl = imageUrl.toString())
         }
     }
+    private fun pageListParse(document: Document): List<Page> {
+        return document.select(pageUrlSelector).mapIndexed { index, element ->
+            Page(index, document.location(), element.absUrl("src"))
+        }
+    }
+    private fun extractScriptData(document: Document): String {
+        return document.select("script").map(Element::data)
+            .firstOrNull(pageRegex::containsMatchIn)
+            ?: throw Exception("Failed to load pages: Script data not found")
+    }
+
+    private fun extractJsonContent(scriptData: String): String {
+        return pageRegex.find(scriptData)
+            ?.groups?.get(1)?.value?.replace("\\", "")
```

Comment:
Maybe parse json with the json library? Like:
https://github.com/keiyoushi/extensions-source/blob/160efb27c66e7cf29cc85b293954d5b64ec13503/src/en/arvenscans/src/eu/kanade/tachiyomi/extension/en/arvenscans/VortexScans.kt#L30